### DISCUSSION TITLE
Web3 Unleashed contract test failed.

### DISCUSSION THREAD

**Discussion by way2ex** [ConsenSys-archive/truffle#5936] [Web3 Unleashed]
I have read the web3-unleashed docs and follow the guide to prepare the contract code and test code. But the test failed.
![image](https://user-images.githubusercontent.com/24694223/221737161-a143d600-b584-4aae-88c4-2208b51002e3.png)

What did i forget to prepare?

**Comment by FlaviuWeb3** [comment]
Hi @zhuqingguang could you please share with me a repository link and the steps to reproduce the issue?

**Reply by way2ex** [reply]
https://github.com/zhuqingguang/tx-snap-issue-demo 
Here is the repo. You can run the following command:
```
yarn
yarn start 
```
and open another terminal:
```
cd packages/truffle && yarn test
```
I init the repo with the blog: https://trufflesuite.com/guides/metamask-snap/#create-nftvault
